x/concentrated-liquidity/tick_test.go

```diff
@@ -96,6 +96,7 @@ func (s *KeeperTestSuite) TestInitOrUpdateTick() {
 		tickExists             bool
 		expectedLiquidityNet   sdk.Dec
 		expectedLiquidityGross sdk.Dec
+		minimumGasConsumed     uint64
```

Comment:
set as testing param, just so if we ever want to change it from constant gas cost to linearly scaling gas cost, we can just change this value in test